configure.py

```diff
@@ -1844,7 +1844,8 @@ def make_bin_override():
 configure_intl(output)
 configure_static(output)
 configure_inspector(output)
-configure_section_file(output)
+if options.node_section_ordering_info != "":
+  configure_section_file(output)
```

Comment:
I believe we still need `o['variables']['node_section_ordering_info'] = ""` by default, so we should probably just move the gold.ld detection logic into the branch inside the definition of `configure_section_file`?